openshift/templates/email-verification-agent/email-verification-agent-deploy.yaml

```diff
@@ -4,14 +4,59 @@ apiVersion: v1
 metadata:
   name: ${NAME}
 objects:
+  - kind: ExternalNetwork
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: ${NAME}${SUFFIX}
+      network: ${NAME}${SUFFIX}
+      labels:
+        name: ${NAME}${SUFFIX}
+        network: ${NAME}${SUFFIX}
+        app: ${APP_NAME}${SUFFIX}
+        app-group: ${APP_GROUP}
+        env: ${TAG_NAME}
+    spec:
+      description: |
+        Define the network parameters for accessing remote agents.
+      entries:
+        - 0.0.0.0/0
+      servicePorts:
+        - tcp/9700:9799
+        - tcp/80
+        - tcp/443
+  - kind: NetworkSecurityPolicy
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: ${NAME}
+      labels:
+        name: ${NAME}
+        env: ${TAG_NAME}
+        app: ${APP_NAME}${SUFFIX}
+        app-group: ${APP_GROUP}
+        env: ${TAG_NAME}
+    spec:
+      description: Allow the the agent to access the service and external network
+      source:
+        - - $namespace=${NAMESPACE_NAME}-${TAG_NAME}
+          - role=${ROLE}
+          - app=${APP_NAME}${SUFFIX}
+          - env=${TAG_NAME}
+      destination:
+        - - ext:name=${NAME}
```

Comment:
Missing ${SUFFIX}, so the reference will not work.

Reply:
fixed